new-docs/content/any/getting-started/policies/index.md

```diff
@@ -9,325 +9,401 @@ description: Learn about writing Oso policies - the source of truth for authoriz
-head:
+Here we've recoded the implicit disjunction expressed by three rules
+as an explicit disjunction of conditions using the logical `or` operator.
+(As you might guess, there are also logical `and` and `not` operators.
+Like `or`, they may only be used in rule bodies, i.e., after an `if`.)
 
+Yet another way to write this policy would be to use the `in` operator:
 ```polar
-allow(_actor: Admin, "approve", _report);
+allow(_actor, action, _resource) if
+    action in ["GET", "POST", "DELETE"];
 ```
 
-The rule will fail when evaluated on a regular `User` and succeed when
-evaluated on an `Admin`, encoding an implicit `if Admin` condition.
-
-This is another example of the rule matching process: instead of matching
-against a concrete value, we are instead checking to make sure the type of the
-input matches the expected type — in this case, an `Admin`.
-
-{{% callout "Tip" "green" %}}
-  Try to use type specializers as often as possible. It will help make sure you
-  don't accidentally allow access to an unrelated resource which happens to
-  have matching fields.
-{{% /callout %}}
+Here the list of allowed actions is encoded as a literal Polar list
+(comma separated terms between square brackets). The `in` operator
+is a membership check: it succeeds when its left-hand side (`action`)
+appears anywhere in its right-hand side (the list `["GET", "POST",
+"DELETE"]`), which is true just when the left-hand side equals the
+list's first element, *or* its second element, *or* its third
+element, .... So this rule means the same thing as the ones above.
 
-### Combining Rules
+## Classes, Instances, and Fields
 
-Rules can be thought of as equivalent to methods in imperative programming. The
-same idea should be applied when writing policies: any piece of logic that you
-want to reuse throughout a policy can be extracted out into a new rule.
+In our example app, an action is represented by a string, but the
+resources we're protecting are instances of our `Expense` model class.
+Let's use its `submitted_by` field to implement the sample rule:
 
-The benefits of that extraction are that it makes it easier to keep logic
-consistent throughout and often results in a much more readable policy.
+* A user may read any `Expense` that they submitted.
 
-Take the following example. We want a rule saying that accountants can read
-expenses. Our initial version might look like:
+One way to write such a rule in Polar is like this:
 
 ```polar
-allow(user: User, "read", expense: Expense) if
-    user.role = "accountant";
+allow(actor, "GET", resource) if
+    resource.submitted_by = actor;
 ```
 
-This would be fine, but if, for example, we wanted to allow the CFO to do
-whatever an accountant can do, we would need to duplicate all the rules. Or if
-we want to change how an application determines roles we would need to change
-all locations using this.
+The Polar term `resource.submitted_by` refers to the `submitted_by`
```

Comment:
Would be good to explicitly say that we can pass application objects into queries, so we can write rules over them. I know we say it before, but good to reinforce what's happening here.